Makefile

```diff
@@ -1,18 +1,21 @@
 DEV_IMAGE_ORG = jboss-dataservices
 DOCKER_REGISTRY_ENGINEERING =
 DOCKER_REGISTRY_REDHAT =
+TEST_IMAGE_NAME = datagrid-online-services:dev
 DEV_IMAGE_NAME = datagrid-online-services-dev
 ADDITIONAL_ARGUMENTS =
 
 CE_DOCKER = $(shell docker version | grep Version | head -n 1 | grep -e "-ce")
 ifneq ($(CE_DOCKER),)
-DOCKER_REGISTRY_ENGINEERING = docker-registry.engineering.redhat.com
-DOCKER_REGISTRY_REDHAT = registry.access.redhat.com/
-DEV_IMAGE_FULL_NAME = $(DOCKER_REGISTRY_ENGINEERING)/$(DEV_IMAGE_ORG)/$(DEV_IMAGE_NAME)
-CONCREATE_CMD = concreate generate --overrides=overrides.yaml --target target-docker;
+	DOCKER_REGISTRY_ENGINEERING = docker-registry.engineering.redhat.com
+	DOCKER_REGISTRY_REDHAT = registry.access.redhat.com/
```

Comment:
I was wondering - do we need a tab here? For some reason, adding a tab here disables my Intellij Makefile support. Weird....